### DISCUSSION TITLE
zwave end device range quality declaration based on signalstrength in dbm

### DISCUSSION THREAD

**Discussion by priyanka043** [zwave-js/zwave-js-ui#3513] [Q&A]
Hi,

 How to describe the zwave end device signal strength i.e( poor, average, good, excellent) based on signalstrength in dbm, can anyone help me with ranges like for zigbee we follow as following

Excellent Signal Strength:

RSSI: Typically between -30 dBm to -50 dBm.

Good Signal Strength:

RSSI: Usually between -51 dBm to -60 dBm.

Average Signal Strength:

RSSI: Ranges from -61 dBm to -70 dBm.

Poor Signal Strength:

RSSI: Typically between -71 dBm to -80 dBm.

Very Poor Signal Strength:

RSSI: Beyond -80 dBm.

**Comment by robertsLando** [comment]
Where do you want this to be shown on UI exactly?

**Reply by priyanka043** [reply]
Actually at our end we want to describe it based on rssi values that we get from the device.

**Reply by robertsLando** [reply]
My question is, the rssi is actually specific to a specific route, you already have an idea about the quality by checking the edge color in network graph
